packages/cli/generators/discover/index.js

```diff
@@ -0,0 +1,292 @@
+path = require('path');
+const fs = require('fs');
+const ArtifactGenerator = require('../../lib/artifact-generator');
+const modelMaker = require('../../lib/model-discoverer');
+const debug = require('../../lib/debug')('discover-generator');
+const utils = require('../../lib/utils');
+const modelDiscoverer = require('../../lib/model-discoverer');
+
+module.exports = class DiscoveryGenerator extends ArtifactGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+
+    this.option('dataSource', {
+      type: String,
+      alias: 'ds',
+      description: 'The name of the datasource to discover',
+    });
+
+    this.option('views', {
+      type: Boolean,
+      description: 'Boolean to discover views',
+      default: true,
+    });
+
+    this.option('schema', {
+      type: String,
+      description: 'Schema to discover, required for postgresql',
+      default: 'public',
+    });
+
+    this.option('discoverAll', {
+      type: Boolean,
+      description:
+        'Avoids the prompt asking for specific models and just discovers them all',
```

Comment:
Is `Discover all models without prompting users to select` a better description?

Reply:
Yes, that's much better.